src/pages/Talents.tsx

```diff
@@ -34,12 +40,22 @@ function Talents() {
         })}
       </div>
 
-      {users.length === 20 && (
-        <a
-          href='#'
-          className='px-5 py-2  border border-indigo-600 rounded-full text-indigo-600 hover:text-white hover:bg-indigo-700'>
-          Load More
-        </a>
+      {users.length > 0 && !noMoreData ? (
+        <div className='flex justify-center items-center gap-10 flex-col pb-5'>
+          <button
+            type='submit'
+            className={`px-5 py-2 mt-5 content-center border border-indigo-600 rounded-full text-indigo-600 
+              hover:text-white hover:bg-indigo-700
+            `}
+            disabled={noMoreData}
+            onClick={() => loadMore()}>
```

Comment:
Didn't have any issue with double or triple click etc.
In each case only one request was made.